cmd/mt-fakemetrics/cmd/datafeed.go

```diff
@@ -161,6 +162,19 @@ func dataFeed(outs []out.Out, orgs, mpo, period, flush, offset, speedup int, sto
 		panic(fmt.Sprintf("num-unique-custom-tags must be a value between 0 and %d, you entered %d", len(customTags), numUniqueCustomTags))
 	}
 
+	vp, err := policy.ParseValuePolicy(valuePolicy)
```

Comment:
This might be personal taste, but i'd prefer to pass the parsed policy into `dataFeed()` as an argument, instead of directly using the global variable from here. Otherwise this makes it unnecessarily hard to understand the behavior of `dataFeed()`

Reply:
It's a global/root flag, that's why it is in `root` and doesn't get parsed in the caller functions. If you look at `feed` or `backfill` you will see that they duplicate code / settings individually. Since this settings applies to everything, I have it broken out this way. Otherwise we have to parse it in each of those other places which may or may not use it. The better question is why `feed` and `backfill` duplicate all those settings instead of just setting them in `root`.

Reply:
> The better question is why feed and backfill duplicate all those settings instead of just setting them in root.

because there are many other commands than just backfill and feed, to which settings that only apply to backfill and feed, do not apply.

dataFeed is a low level worker function, it should be passed in parameters, and not be parsing user input.